modules/terraform-aci-endpoint-loop-protection/variables.tf

```diff
@@ -1,12 +1,13 @@
-variable "action" {
-  description = "Action. Choices: `bd-learn-disable`, `port-disable`."
-  type        = string
-  default     = "port-disable"
+variable "bd_learn_disable" {
+  description = "Action: BD Learn Disable."
+  type        = bool
+  default     = false
```

Comment:
this is different than the default that we specified in defaults.yaml

Reply:
Generally the low level modules default to the APIC default (`port-disable`). The high level NAC module leverages a defaults file which aligns with Cisco recommended configuration (`bd-learn-disable`), or to ensure backwards compatibility.